index.js

```diff
@@ -72,11 +72,34 @@ I18nPlugin.prototype.apply = function(compiler) {
 
 /**
  *
- * @param {object}	localization
+ * @param {object}  localization
+ * @param {string}  string key
+ * @returns {*}
+ */
+function byString(object, stringKey) {
+    stringKey = stringKey.replace(/\[(\w+)\]/g, '.$1'); // convert indexes to properties
```

Comment:
Either don't allow this at all or implement it correctly.

`abc[def]` is not the correct syntax. `abc["def"]` is the correct syntax. Also make sure that `abc["def['ghi']"]` works ;).